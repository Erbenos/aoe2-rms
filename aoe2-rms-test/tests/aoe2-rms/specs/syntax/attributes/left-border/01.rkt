#lang aoe2-rms

<LAND-GENERATION>
(create-player-lands
 (terrain-type 'DIRT)
 (land-percent 100)
 (left-border 60))
