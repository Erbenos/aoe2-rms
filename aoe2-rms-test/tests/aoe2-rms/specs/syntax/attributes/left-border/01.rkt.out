<LAND_GENERATION>
create_player_lands {
  terrain_type DIRT
  land_percent 100
  left_border 60
}
